Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerStats.h

```diff
@@ -8,10 +8,12 @@
 
 #pragma once
 
-#include <AzCore/Time/ITime.h>
-#include <AzCore/std/containers/vector.h>
+#include <AzCore/EBus/ScheduledEvent.h>
 #include <AzCore/std/containers/array.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/Time/ITime.h>
 #include <Multiplayer/MultiplayerTypes.h>
+#include <Multiplayer/MultiplayerStatSystem.h>
```

Comment:
The `ScheduleEvent.h`, `ITime.h` and `MultiplayerStatsSystem.h` includes don't look used in this file. Are they?

Reply:
I've cleaned this up.